### PR TITLE
Powershell Version of Binary Search

### PR DESCRIPTION
**Pull request by itadder**
First Commit of a PowerShell version of Binary Search. Please let me how it can be improved.
This is my first attempt to port the algorithms in the book to Powershell. Hope this helps others, I will on the others as I go on in the book in Powershell. Since PowerShell is now open source, and available to others in Linux, mac, and windows it should work cross platform.

### CODE REVIEW

File: 01_introduction_to_algorithms/PowerShell/01_binary_search/Search-Binary.ps1 [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        
+        $item
+    )
+
+    [int] $low = 0;
+    $high = $list.Length -1;
+
+    while ($low -lt $high) {
+
+        $mid = ($low + $high);
```
Comment (egonSchiele) [inline 01_introduction_to_algorithms/PowerShell/01_binary_search/Search-Binary.ps1]: This should be `$mid = ($low + $high) / 2;`. The code is wrong in the book (it is a known bug).

**Comment by egonSchiele** [comment]
Hi,
I don't know powershell, but I'm happy to merge this. I did catch an error in the code (added a comment).

**Comment by itadder** [comment]
I made the change to to reflect $mid = ($low + $high) / 2;.

**Comment by egonSchiele** [comment]
Thanks!

**Pull request merged**